cookingWebsite/yourCookbook/viewRecipe.php

```diff
@@ -0,0 +1,112 @@
+<?php
+include('config/init.php');
+verifyUserCooking();
+$recipeId = $_REQUEST['recipeId'];
+$loginId = $_SESSION['loginId'];
+echo yourCookbookEchoHtmlHeader("New Recipe");
+?>
+<script>
+function addTagExistingRecipe(recipeId){
+    var recipeIngredientName = $('#recipeIngredientName').val();
+    var submitItem = $('#submitItem').val();
+    console.log(recipeId, submitItem, recipeIngredientName)
+    $.post('/ajax/saveAddedtag.php', {recipeIngredientName:recipeIngredientName, submitItem:submitItem, recipeId:recipeId}, function(Data){
+          if (Data == "failure"){
+            alert("Error");
+            return false;
+        }
+        // $('.addTag').html(Data)
+    });
+    alert("The tag was sucessfully added!");
+    return false;
+}
+function addRating(loginId, recipeId){
+    var rating = $('#rating').val();
+    var submitItem = $('#submitItem').val();
+    console.log(recipeId, submitItem, loginId)
+    $.post('/ajax/saveRating.php', {rating:rating, submitItem:submitItem, recipeId:recipeId, loginId:loginId}, function(Data){
+          if (Data == "failure"){
+            alert("Error");
+            return false;
+        }
+    });
+    alert("The tag was sucessfully added!");
+    return false;
+}
+function deleteTagExistingRecipe(recipeId){
+    var recipeIngredientName = $('#recipeIngredientName').val();
+    var deleteItem = $('#deleteItem').val();
+    $.post('/ajax/deleteTagExistingPost.php', {recipeIngredientName:recipeIngredientName, deleteItem:deleteItem, recipeId:recipeId}, function(Data){
+          if (Data == "failure"){
+            alert("Error");
+            return false;
+        }
+        // $('.deleteTag').html(Data)
+    });
+    alert("The tag was sucessfully removed!");
+    return false;
+}
+</script>
+<br/><br/><div class = 'recipeHeader'><span class='recipe'>
+
+<?php
+echo recipeContent($recipeId, 'title');
+?>
+ <!-- <a href='#' id='faves' onclick='favorited()'><i class="fa fa-star" aria-hidden="true"></i></a> -->
+</div><p style='text-align:center; font-size:14px'>By:
+<?php
+    echo recipeContent($recipeId, 'firstName')." ".recipeContent($recipeId, 'lastName');
```

Comment:
Also, I'm not sure a firstName should be stored with each recipe. It seems like it'd be better to store a userId which links to a user record that has the person's name. Otherwise when a user updates their name, their old recipes would have the wrong data listed. It's generally best to avoid duplicating information in the db because it makes it hard to keep things up-to-date.